docs/blog/2025-03-19-using-angular-with-rspack.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'Using Angular with Rspack'
+date: 2025-03-19
+slug: using-angular-with-rspack
+authors: [Colum Ferry]
+tags: [angular, webpack, rspack, nx]
+cover_image: /blog/images/2025-03-19/angular-rspack-logo-small.png
+description: Learn about how and why to use Rspack with Angular thanks to Nx's efforts for supporting Rspack for Angular.
+---
+
+Building applications with [Angular](https://angular.dev) has always been a lesser concern for most Angular developers due to the abstractions that Angular created called `builders`. The underlying implementation details were hidden from the developer who only needed to run either `ng build` or `nx build`.  
+Despite this, most Angular developers knew that it was originally [webpack](https://webpack.js.org/) that was used to build their applications. This was a great solution at the time and it was even possible to extend their builds by leveraging custom webpack configurations and plugins.
+
+Over time, as applications grew in size and complexity, it became clear that the inherit slowness with Webpack build speeds was becoming more and more of an issue for Angular developers.
```

Comment:
>  build speed issue by building out a new build pipeline

😅 idk why but this reminded me of https://youtube.com/clip/UgkxI9oC4W7_NCEFIekznXVdOz2EyE6nNbEr?si=qWSBM0R3o8WShbCP